packages/core/src/App/Containers/Modals/mt5-notification/mt5-notification-description.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { isMobile } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+
+const MT5NotificationDescription = () => {
```

Comment:
```suggestion
const MT5NotificationDescription = () => {
    <MobileWrapper>
```
follow what aizad said